ketcher-autotests/tests/test-data/Rxn-V2000/benzene-chain-reaction.rxn

```diff
@@ -68,4 +68,8 @@ $MOL
 
   1  0  0     0  0            999 V2000
    25.8000   -6.3500    0.0000 Br  0  0  0  0  0  0  0  0  0  0  0  0
+<<<<<<< HEAD
```

Comment:
It looks like unresolved merge conflict.. Is it expected?

Reply:
resolved